.github/workflows/alpha-release.yml

```diff
@@ -31,6 +31,7 @@ jobs:
     name: Run publish script
     runs-on: ubuntu-latest
     needs: [ test ]
+    environment: deloyment
```

Comment:
what's the specific need for this?

Reply:
I added the secret only for the environment called `deployment`